UI/js-src/lsmb/login.js

```diff
@@ -1,4 +1,5 @@
 /** @format */
+/* eslint global-require: 0 */
```

Comment:
We could put require on the toplevel by voing the 2 requires into a single toplevel statement. Isn't that "nicer" ?

Reply:
The rule expect only 1 require par module, at the top of the file. We have 3 in login.js